README.md

```diff
@@ -1,59 +1,25 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# HelloMicroverse
 
-> One paragraph statement about the project.
+> Hello Microverse is the project that made for the first code review of the program. 
 
 ![screenshot](./app_screenshot.png)
 
 Additional description about the project and its features.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo
-
-[Live Demo Link](https://livedemo.com)
-
-
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
-
-### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
-
-
+- HTML & CSS
+- GitHub Workflows
```

Comment:
Kindly fill information removed like how to download your repo, install your project locally how to use it which requirements are needed to keep on mind before to install. Probably testing and deployment isn't required.

Reply:
Hello!